pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -200,7 +200,11 @@ func classifyNodes(npm NodePodsMap, thresholds api.ResourceThresholds, targetThr
 			klog.V(2).Infof("Node %#v is over utilized with usage: %#v", node.Name, usage)
 			targetNodes = append(targetNodes, nuMap)
 		} else {
-			klog.V(2).Infof("Node %#v is appropriately utilized with usage: %#v", node.Name, usage)
+			if nodeutil.IsNodeUnschedulable(node) {
+				klog.V(2).Infof("Node %#v is unschedulable", node.Name)
```

Comment:
could you still add the usage log here too? May be helpful to see if you have unschedulable nodes that are underutilized

Reply:
Changing `Node %#v is appropriately utilized with usage` to `Node %#v is utilized with usage`.